include/ck/utility/reduction_operator.hpp

```diff
@@ -361,7 +432,7 @@ struct InMemoryDataOperationSupportedOnDataType<InMemoryDataOperationEnum::Add,
     static constexpr bool value =
         is_same<DataType, float>::value || is_same<DataType, double>::value ||
         is_same<DataType, half_t>::value || is_same<DataType, int8_t>::value ||
-        is_same<DataType, int32_t>::value;
+        is_same<DataType, int32_t>::value || is_same<DataType, f8_t>::value;
```

Comment:
We dont support Add for fp8 in include/ck/utility/dynamic_buffer.hpp:141. It should be probably implemented. Could you verify if our test catch such problem? I believe due to lack of such support we should get wrong values.

Reply:
Btw I think Add is not needed for avg and max pool.

Reply:
We do support load/store https://github.com/ROCm/composable_kernel/blob/develop/include/ck/utility/amd_buffer_addressing.hpp#L429 so the only thing to do is just add two f8_t values together, but we would have to do it through fp32 casting.